mkdocs.yml

```
@@ -80,8 +80,10 @@ nav:
         GAMMA Initiative: mesh/gamma.md
         Service Facets: mesh/service-facets.md
     - Implementations:
-        List: implementations.md
-        Comparison: implementation-table.md
+        - List: implementations.md
+        - Comparison: implementation-table-v1.1.0.md
+        - Past Version Comparisons:
+            - v1.0.0: implementation-table-v1.0.0.md
```

Maybe something like this to line up with my comment from above
```suggestion
        - Comparisons: 
            - v1.1: implementations/v1.1.md
            - v1.0: implementations/v1.0.md
```